src/main/java/model/item/Equipment.java

```diff
@@ -1,50 +1,56 @@
 package model.item;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.ReadOnlyIntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
 import manager.Database;
 import manager.TypTranslation;
+import manager.Utility;
 import model.Currency;
 import model.upgrade.Upgrade;
 import model.upgrade.UpgradeFactory;
 
 import java.util.*;
+import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
 public abstract class Equipment extends Item {
 
     protected static Random rand = new Random();
 
-    protected String material = "";
-    protected int slots = 0;
-    protected ArrayList<Upgrade> upgrades = new ArrayList<>();
-
-    public String getMaterial() {
-        return material;
-    }
-
-    public void setMaterial(String material) {
-        this.material = material;
-    }
-
-    public int getSlots() {
-        return slots;
-    }
-
-    public void setSlots(int slots) {
-        this.slots = slots;
-    }
-
-    public Collection<Upgrade> getUpgrades() {
-        return upgrades;
+    protected String material;
+    protected int slots;
+    protected ArrayList<Upgrade> upgrades;
+    protected IntegerProperty wear;
+    protected IntegerProperty wearStep;
+    protected Collection<Consumer<Equipment>> onBreakListeners;
+
+    public Equipment() {
+        super();
+        this.material = "";
+        this.slots = 0;
+        this.upgrades = new ArrayList<>();
+        this.wearStep = new SimpleIntegerProperty(0);
+        this.wear = new SimpleIntegerProperty(0);
+
+        int neededSteps = Utility.getConfig().getInt("character.wear.stepsNeeded");
+
+        if (neededSteps > 0) {
+            this.wear.bind(wearStep.divide(neededSteps));
+        }
+        this.onBreakListeners = new ArrayList<>();
     }
 
-    public void setUpgrades(ArrayList<Upgrade> upgrades) {
-        this.upgrades = upgrades;
-        this.upgrades.sort(Comparator.comparing(Upgrade::getFullName));
-    }
+    protected abstract boolean shouldBreak();
 
-    public String upgradesAsString() {
-        return this.upgrades.stream().map(Upgrade::getFullName).collect(Collectors.joining(", "));
+    public void onUse() {
+        this.setWearStep(getWearStep() + 1);
+        if (shouldBreak()) {
+            for (Consumer<Equipment> listener : this.onBreakListeners) {
+                listener.accept(this);
+            }
+        }
```

Comment:
onUse suggests, that there might be different effects on the Equipment in the future. If that is the case, this whole wear part should be another, private, method.
Otherwise, onUse() should be called applyWear() or something. 

Also, make the +1 a int parameter